src/core/lib/support/ref_counted_ptr.h

```diff
@@ -76,6 +76,19 @@ class RefCountedPtr {
   T& operator*() const { return *value_; }
   T* operator->() const { return value_; }
 
+  bool operator==(const RefCountedPtr& other) const {
+    return value_ == other.value_;
+  }
+  bool operator==(T* other) const {
```

Comment:
you probably want to make the argument be `const T* other`. Same on line 88

Reply:
Done.